configurator/templates/AppServer/configs.ini

```diff
@@ -0,0 +1,28 @@
+[SETTINGS]
+read_env_variables=false
+server=AppServer
+pack_location=/Users/anuruddha/Downloads/wso2as-5.2.1/repository/conf
```

Comment:
Few concerns:
- The variable name "pack_location", shall we rename this to something like "distribution_file_path"?
- Shall we keep the value empty? Currently it is containing a file path specific to your your environment.